build.gradle

```diff
@@ -24,6 +24,11 @@ repositories {
     jcenter()
 }
 
+java {
+    sourceCompatibility = "1.8"
+    targetCompatibility = "1.8"
```

Comment:
I think this works, but there's an option to explicitly set the javac `--release` flag (which is exactly what is needed for the bytecode to be compatible with a specific version):

https://docs.gradle.org/current/dsl/org.gradle.api.tasks.compile.CompileOptions.html#org.gradle.api.tasks.compile.CompileOptions:release

I believe that it only works if `sourceCompatibility` is also set to the same version because otherwise the IDE may not pick up when you use Java 9+ stuff and it will then fail compilation later.

Anyway, I think that if the jar is currently runnable  on Java 8 , then it looks like `targetCompatibility` is still working, so this should be fine.